plugins/chat/assets/javascripts/discourse/components/chat-header-icon.hbs

```diff
@@ -1,11 +0,0 @@
-<a
-  href={{this.href}}
-  tabindex="0"
-  class={{concat-class "icon btn-flat" (if this.isActive "active")}}
->
-  {{d-icon "d-chat"}}
-
-  {{#unless this.currentUserInDnD}}
-    <ChatHeaderIconUnreadIndicator />
-  {{/unless}}
-</a>
```

Comment:
We can move this file using `git mv` instead of deleting it and creating the new file:

```
mkdir plugins/chat/assets/javascripts/discourse/components/chat/header
git mv plugins/chat/assets/javascripts/discourse/components/chat-header-icon.hbs plugins/chat/assets/javascripts/discourse/components/chat/header/icon.hbs
```

This will preserve the history of this file, it will be easier to navigate in the future. Also, the diff here will be more convenient for reviewing, the file will be shown as moved and since there are also some updated lines, those lines will be highlighted.

Reply:
Yes I have been quite lazy on doing this :p I guess it's probably a good idea to do it

Reply:
Thank you for looking at that!